src/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.Counters.cs

```diff
@@ -2,33 +2,37 @@ namespace System.Diagnostics.Tracing
 {
     public abstract partial class DiagnosticCounter : System.IDisposable
     {
-        internal DiagnosticCounter(string name, System.Diagnostics.Tracing.EventSource eventSource) { }
+        internal DiagnosticCounter() { }
+        public string? DisplayName { get { throw null; } set { } }
+        public string? DisplayUnits { get { throw null; } set { } }
```

Comment:
There's a PR in coreclr to change this.

Reply:
Thanks for the heads-up. Anyways, before merging this PRs I'm addressing feedback and re-running GenAPI against latest coreclr to make sure everything is up to date.